MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include LICENSE
```

Comment:
The napari cookiecutter example plugin is set up with:
* pyproject.toml
* setup.cfg (no setup.py file, most of that info is moved into setup.cfg)
* MANIFEST.in
* and a napari.yaml file.

Both the pyproject.toml and MANIFEST.in files are exactly the same as the [cookiecutter plugin example](https://github.com/napari/cookiecutter-napari-plugin).
I replaced the current setup.py file with a setup.cfg, and transferred the information across.